CMakeLists.txt

```diff
@@ -9,6 +10,8 @@ option(BUILD_GTEST "Builds the googletest subproject" OFF)
 #Note that googlemock target already builds googletest
 option(BUILD_GMOCK "Builds the googlemock subproject" ON)
 
+cmake_policy(SET CMP0048 NEW)
```

Comment:
I will try what happens when I remove it. But I think I got an warning or error without it.